Profile Folder/chrome/locale/pl-PL/properties/gsettings.properties

```diff
@@ -1,15 +1,15 @@
-geckiumSettings=[!!! Gèçƙïú₥ §èƭƭïñϱƨ !!!]
+geckiumSettings=Ustawienia Geckium
 
 addAnApp=Dodaj aplikację
 
 themeHasNoDescription=Ten motyw nie posiada opisu.
-builtInTheme=[!!! [ßÛÌ£T-ÌN] {{themeName}} !!!]
-showInFolder=[!!! §λôω ïñ ƒôℓδèř !!!]
-apply=[!!! Âƥƥℓ¥ !!!]
-yes=[!!! Ýèƨ !!!]
+builtInTheme=Wbudowany {{themeName}}
```

Comment:
"Built-in" is meant to be separate from the `themeName`, the way it's written here makes it seem like "Built-in" is part of the `themeName`.
English example:
![image](https://github.com/user-attachments/assets/c373368f-c3ba-4974-9109-e59d084a104c)

Reply:
oh i understand it now. did the change (never used geckium in en-us since pl-pl is a thing)